creme/creme_config/tests/transfer/test_importing.py

```diff
@@ -1311,7 +1311,7 @@ def test_customfields02(self):
         response = self.assertPOST200(self.URL, data={'config': json_file})
         self.assertFormError(
             response, 'form', 'config',
-            _('This custom-field type is invalid: {}.').format(1024)
+            _('This custom-field type is invalid: {}.').format(self.UNUSED_PK)
         )
```

Comment:
Here the custom field type corresponds to a constants in the code, not a PK which could be reached if sequences at not reset. It works but the name 'PK' if not totally accurate (I have no better proposition sorry). Hummm

Reply:
I could use a test-specific value here and first check that it's not an available choice of the formfield ?

Reply:
Doesn't seem trivial with the ImportForm

Reply:
yes of course (leave 1024 or add a named variable/attribute)